Formula/ice.rb

```diff
@@ -58,6 +58,7 @@ def install
     args = %W[
       prefix=#{prefix}
       USR_DIR_INSTALL=yes
+      SLICE_DIR_SYMLINK=yes
```

Comment:
Hmmm.... if you need non-symlinked directories, usually people use `#{var}`, `#{var}/run`, `#{var}/foo`, `#{var}/run/foo`, `#{etc}`, or `#{etc}/foo`, since `#{var}` is `#{HOMEBREW_PREFIX}/var` not `#{prefix}/var` (i.e., it's `/usr/local/var` not `/usr/local/Cellar/foo/1.2.3/var`) and `#{etc}` is `#{HOMEBREW_PREFIX}/etc`, not `#{prefix}/etc` (i.e., it's `/usr/local/etc` not `/usr/local/Cellar/foo/1.2.3/etc`).